runtime/src/accounts_index.rs

```diff
@@ -1161,18 +1161,18 @@ impl<T: 'static + Clone + IsCached + ZeroLamport> AccountsIndex<T> {
         account_indexes: &HashSet<AccountIndex>,
     ) {
         let roots_tracker = &self.roots_tracker.read().unwrap();
-        let max_root = Self::get_max_root(&roots_tracker.roots, &list, max_clean_root);
+        let max_present_root = Self::get_max_root(&roots_tracker.roots, &list, max_clean_root);
```

Comment:
nits: can we rename `max_present_root` to `max_root_in_slot_list`, and `get_max_root()` to `get_max_root_in_slot_list()` to distinguish these are different than the absolute max root?

Reply:
I like it! Will do.

Reply:
Done!